src/components/Comments/Comment.js

```diff
@@ -169,6 +173,19 @@ const Comment = ({
     }
   };
 
+  const onPressDm = async () => {
+    try {
+      refSheet.current.open();
+      setLoading({...loading, loadingGetAllowAnonDmStatus: true});
+      const data = await getAllowAnonDmStatus('comment', comment?.id);
+      setUserAllowDm(data?.user.allow_anon_dm);
+    } catch (e) {
+      console.log(e);
+    } finally {
+      setLoading({...loading, loadingGetAllowAnonDmStatus: false});
+    }
+  };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new function `onPressDm` has been added which fetches whether the user allows anonymous DMs or not. This is a good addition as it provides more control over the DM feature based on user preferences.



<!-- This is an auto-generated comment by OSS CodeRabbit -->